### PR TITLE
Add property

### PR DESCRIPTION
**Pull request by greg-Da**

### CODE REVIEW

File: src/pages/Register.jsx [hunk 1]
```diff
@@ -56,41 +60,66 @@ export default function Register() {
...
-        </small>
-      </Card>
+    <div className="absolute flex w-full h-full bg-gray-300">
+      <div className="m-auto w-5/6 md:w-4/6">
+        <Card className="p-6">
+          <h1 className="text-center mb-5 font-bold text-3xl">Register</h1>
+          <form className="grid gap-y-5">
+            <input
+              className="rounded-full border p-2 outline-none"
+              value={firstName}
```
Comment (Creiwry) [inline src/pages/Register.jsx:70]: First name and last name are not submitable on devise register. I might just remove them all together from the model to be honest, I think we only need a page to change the email address and password

File: src/pages/Property/New.jsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            />
+            Basement
+          </label>
+
+          <label>
+            <input
+              type="checkbox"
+              className="mr-3 w-4 h-4"
+              checked={terrasse}
+              onChange={() => setTerrasse(!terrasse)}
```
Comment (Creiwry) [inline src/pages/Property/New.jsx:164]: It's "Terrace" in English

**Review by Creiwry — CHANGES_REQUESTED** [review]

File: src/pages/Property/New.jsx (hunk 2 above)
Comment (greg-Da) [reply src/pages/Property/New.jsx:164]: ahah sorry I ll change it

**Review by Creiwry — APPROVED** [review]
awesome

**Pull request merged**